io.github.jacalz.rymdport.yml

```diff
@@ -0,0 +1,48 @@
+app-id: io.github.jacalz.rymdport
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+    - org.freedesktop.Sdk.Extension.golang
+command: rymdport
+
+finish-args:
+    - --share=ipc # Share IPC namespace with the host (necessary for X11).
+    - --share=network
+    - --socket=x11
+    - --device=dri # OpenGL rendering support.
+
+    # The below are commented out until there is out of the box support for wayland.
+    #- --socket=fallback-x11 # Only create the x11 socket if wayland isn't avaliable.
+    #- --socket=wayland
+
+    # Needed to support desktop notifications.
+    - --talk-name=org.freedesktop.Notifications
+
+    # Support only the most common directories.
+    - --filesystem=xdg-desktop
+    - --filesystem=xdg-documents
+    - --filesystem=xdg-download
+    - --filesystem=xdg-music
+    - --filesystem=xdg-pictures
+    - --filesystem=xdg-videos
```

Comment:
I think it's fine to use `--filesystem=home` here, but it's up to you. Ideally it should use filesystem portal (e.g. GtkNativeFileChooser does that by default for GTK apps) but I don't know if it's possible to achieve with some Go library.

Reply:
Thanks. I might switch that over to `—filesystem=home` then. What I have now works but it prints an error every time the file viewer is opened.

I would want to get the portals working through Dbus (or CGO by linking to the C-library but I want try and avoid that) but there isn’t a lot of information or examples out there (especially for Dbus).